raiden/ui/cli.py

```diff
@@ -589,6 +589,7 @@ def _run_smoketest():
                 else:
                     success = True
             finally:
+                app.raiden.get()
```

Comment:
Ops.. `get` should be after `stop`, stop tells raiden to gracefully exit, `get` waits for it and re-raises any exception that may have been raised.